.github/workflows/pull-request.yml

```
@@ -41,6 +41,13 @@ jobs:
           TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
           NEXT_TELEMETRY_DISABLED: 1
 
+      - name: Run prettier
```

Nit:
For being consistent, shall we make the first letter uppercase?
```suggestion
      - name: Run Prettier
```